api/Dockerfile

```
@@ -0,0 +1,25 @@
+# syntax=docker/dockerfile:1
+
+ARG GO_VERSION=1.25.3
```

This can be updated now (https://github.com/moby/moby/pull/51418):

```suggestion
ARG GO_VERSION=1.25.4
```